rust/lang/security/audit/path/path-injection-cli-arg.yaml

```diff
@@ -0,0 +1,35 @@
+rules:
+  - id: path-injection-cli-arg
+    patterns:
+      - pattern-either:
+          - patterns:
+              - pattern: |
+                  fn $FUNC(...) {
+                    let $VAR = $F_ENV(...);
+                    ...
+                    $X.push(...);
+                    ...
+                    $F_FS($X);
+                  }

```

Comment:
Should be a taint mode rule! https://semgrep.dev/docs/writing-rules/data-flow/taint-mode/